docs/pages/desktop-access/getting-started.mdx

```diff
@@ -314,6 +314,59 @@ Computer Configuration > Policies > Administrative Templates > Windows Component
   ![Disable Require](../../img/desktop-access/disable.png)
 </Figure>
 
+### Configure certificate for RDP connections
```

Comment:
Yes, you're right. I've added a sentence to the admonition at the beginning of that chapter that states that it is only necessary for Windows Server 2012 R2. Do you think it is enough or should I add it more explicitly?

Reply:
@ptgott can advise us on this

Reply:
@LKozlowski it would also be good to confirm that if you make these changes and try to run with an old (openssl) version of Teleport that it still works.

Reply:
@zmb3 it works on both version with new certificates